tests/ballerina-compiler-api-test/src/test/resources/test-src/symbols/enum_decl_symbol_test.bal

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Color Enum
+enum Color {
+    RED,
+    BLUE,
+    GREEN
+}
+
+# Kind Enum
+@enumdDecl
+public enum Kind {
+    ONE,
+    # This kind two
+    TWO,
+    @enumMemberDecl
+    THREE
+}
+
+# All Continents
+@enumdDecl
+public enum Continents {
+    ASIA = "asia",
+    @enumMemberDecl
+    AMERICA = "america",
+    # Australia
+    @enumMemberDecl
+    AUSTRALIA = "australia",
+    EUROPE,
+    # Other
+    OTHER
```

Comment:
Shall we change one of these so that the RHS expr is a const ref? And test whether the correct symbol is returned for such cases?